pom.xml

```diff
@@ -36,6 +36,8 @@
         <!-- WSS4J 2.1.x requires Java 7. We need to stay on 2.0.x as long as we need to be Java 6 compatible -->
         <wss4j.version>2.0.5</wss4j.version>
 
+        <org.easytesting.fest.assert.core.version>2.0M10</org.easytesting.fest.assert.core.version>
```

Comment:
Vi overstyrer ikke, vi bare setter versjonen øverst i pom. Det var visst god skikk. Vi lurte på hva M stod for, og kan godt endre versjonen om ønskelig. Det var bare samme som ble brukt i Sdp-klienten.